app/src/main/cpp/DeviceDelegate.h

```diff
@@ -115,6 +121,7 @@ class DeviceDelegate {
   virtual void UpdateHandMesh(const uint32_t aControllerIndex, const std::vector<vrb::Matrix>& handJointTransforms,
                               const vrb::GroupPtr& aRoot, const bool aEnabled, const bool leftHanded) {};
   virtual void DrawHandMesh(const uint32_t aControllerIndex, const vrb::Camera&) {};
+  virtual bool GetTrackedKeyboardInfo(TrackedKeyboardInfo& keyboardInfo) { return false; };
```

Comment:
Naming nit: getters normally return the value they're getting. In this case I'd use a different verb for the method name like Fill, Populate...